ragna/deploy/_ui/central_view.py

```diff
@@ -205,11 +205,17 @@ def on_click_chat_info_wrapper(self, event):
             # `MetadataFilter`s provided as dict
             title = "Metadata Filters"
 
-            metadata_filters_readable = str(
-                MetadataFilter.from_primitive(self.current_chat["metadata"]["input"])
-            ).replace("\n", "<br>")
+            metadata_filters_readable = (
+                str(
+                    MetadataFilter.from_primitive(
+                        self.current_chat["metadata"]["input"]
+                    )
+                )
+                .replace("\n", "<br>")
+                .replace(" ", "&nbsp;")
```

Comment:
Could you explain that?

Reply:
```.replace("\n", "<br>")```
We need to put explicit `<br>` to have line-breaks at the right places, as explained in the other comment.

```.replace(" ", "&nbsp;")```
We need to replace the whitespaces as explicit non-breakable spaces to show indentation, otherwise it's ignored.

Try removing these lines to see how it behaves.

Reply:
So do we even need the `<pre>` tag in that case?

Reply:
Yes, to prevent from any forced line-break, and it also renders with a nice monospaced font.

Here are two screenshots comparing both results
- with `<pre>`
<img width="351" alt="Capture d’écran 2024-12-11 à 12 10 20" src="https://github.com/user-attachments/assets/c07cb9ce-4367-45ab-a98b-95db797c5fbf">

- without `<pre>`
<img width="350" alt="Capture d’écran 2024-12-11 à 12 11 10" src="https://github.com/user-attachments/assets/a3619d5c-7ae5-4136-a82c-ebc4848f70bb">


If you want the non-monospaced font, so no `<pre>`, I can probably fix the result with CSS.